ci/build_common.sh

```diff
@@ -86,6 +86,9 @@ if [ -z ${CCCL_BUILD_INFIX+x} ]; then
     CCCL_BUILD_INFIX=""
 fi
 
+# Where CCCL utility CMake scripts are stored:
+CCCL_CMAKE_SCRIPT_DIR="../cmake"
```

Comment:
suggestion (minor): This script (and all of our scripts) is written in such a way that it guarantees it always executes from the directory where it lives. This is for convenience so we don't have to do things like this and can just use relative paths. So I'd say we can safely remove this.